runtime/src/comm/ofi/comm-ofi.c

```diff
@@ -67,6 +67,14 @@
 #include <rdma/fi_rma.h>
 
 
+////////////////////////////////////////
+//
+// This is the libfabric API version we need.
+//
+
+#define COMM_OFI_FI_VERSION FI_VERSION(1, 8)
```

Comment:
Does this force us into 1.8, or is it 1.8 or newer?

Reply:
It forces us to precisely the 1.8 API.  Among other things the API includes a number of record type definitions, and those can change from minor release to minor release.  The API version you link (static .a) or load (dynamic .so) with has to be the same as the one you compiled with, in order to guarantee that changes in the number, types, or ordering of record members don't cause problems.  This limits our maneuverability with the `CHPL_LIBFABRIC=system` setting, but shouldn't be a problem with the `CHPL_LIBFABRIC=libfabric` setting (i.e., a `third-party/libfabric/install...` installation) once we support the latter.

Reply:
Thanks for the explanation